kw

```diff
@@ -149,6 +149,25 @@ function kw()
         explore "$@"
       )
       ;;
+    man)
+      (
```

Comment:
You have to import kwio.sh here, since you are using complain below. Also, maybe should we move this whole code to a man.sh file and just call a function at it, as we do with other commands? I don't really know.

Reply:
Nice catch! :)

Do you really think that we need to create `man.sh`? I think we can put it in another place, how about `help.sh`?